src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoMend.kt

```diff
@@ -89,8 +102,16 @@ internal object AutoMend : Module(
                     }
                     player.inventory.currentItem = xpSlot
                 }
-                if (autoThrow && player.heldItemMainhand.item === Items.EXPERIENCE_BOTTLE) {
-                    playerController.processRightClick(player, world, EnumHand.MAIN_HAND)
+                if (autoThrow
+                    && throwDelayTimer.tick(throwDelay.value.toLong(), false)
+                    && player.heldItemMainhand.item === Items.EXPERIENCE_BOTTLE) {
+                    if (needsRotationPacket()) {
+                        val packet = PlayerPacket(rotating = true, rotation = Vec2f(player.rotationYaw, 90.0f))
+                        PlayerPacketManager.addPacket(AutoMend, packet)
+                    } else {
+                        playerController.processRightClick(player, world, EnumHand.MAIN_HAND)
+                        throwDelayTimer.reset() // manually reset after every throw only
+                    }
```

Comment:
You need to send the rotation packet every tick

Reply:
Is there a reason? It works perfectly the only issue would be the server tracks the rotations as odd behavior. I'm sending it on the tick before we throw. I don't want the pitch changes to interfere with other behaviors.

Reply:
My only thought is stricter servers like 2b dislike rotation / movement tricks

Reply:
No, it is just changing the rotation and nothing else

Reply:
It was an example

Reply:
> Is there a reason? It works perfectly the only issue would be the server tracks the rotations as odd behavior. I'm sending it on the tick before we throw. I don't want the pitch changes to interfere with other behaviors.

The rotation packet are being send every tick and it will return to normal rotation if you stop sending it in the next tick

Reply:
I hate everything. fixed so it's every tick. So much wasted memory pepeHands